src/medium-zoom.js

```diff
@@ -18,24 +18,32 @@ const isNode = selector => selector && selector.nodeType === 1
  *
  * @param {(string|Element[])} selector The selector to target the images to attach the zoom to
  * @param {object} options The options of the zoom
- * @param {number} [options.margin=0] The space outside the zoomed image
- * @param {string} [options.background="#fff"] The color of the overlay
  * @param {number} [options.scrollOffset=48] The number of pixels to scroll to dismiss the zoom
  * @param {boolean} [options.metaClick=true] A boolean to enable the default action on meta click
  * @param {(string|Element|object)} [options.container] The element to render the zoom in or a viewport object
+ * @param {(string|array)} [options.overlayStyles{background="#fff"}] To pass others css styles to overlay with default background-color=#fff
```

Comment:
"Additional styles to pass to the overlay."

The default value is already documented in here `[options.overlayStyles{ background: "#fff" }]`